lib/db/collections_db.dart

```diff
@@ -5,24 +5,35 @@ import 'package:path/path.dart';
 import 'package:path_provider/path_provider.dart';
 import 'package:photos/models/collection.dart';
 import 'package:sqflite/sqflite.dart';
+import 'package:sqflite_migration/sqflite_migration.dart';
 
 class CollectionsDB {
   static final _databaseName = "ente.collections.db";
-  static final _databaseVersion = 1;
-
   static final collectionsTable = 'collections';
+  static final collectionsTableCopy = 'collections_copy';
```

Comment:
Why do we need this?

Reply:
There is no way to edit a table in SQLite , i have to make a new table and copy the previous table content to it.